firmware/README.md

```diff
@@ -323,6 +332,23 @@ Stillness timer is the length of time the Sensor sees stillness before publishin
 - The integer number of seconds of the current timer value, when e for echo is entered
 - -1: when bad data is entered
 
+### **window_size_set(String)**
+
+**Description:**
+
+Window is the allowed time after the paired door sensor closes in which state 0 can transition to state 1. The default value is 60 minutes. Use this console function to set the window size (time) 
+
+**Argument(s):**
+
+1. The integer number of seconds of the new window timeout.
```

Comment:
This description in the README is correct. But I notice that you have this test in your test plan:

"Enter 10 in the function and confirm that it returns 60, echo again and confirm that it is still 10".

When you enter 10 in the function, I would have expected it to return 10, not 60. How did this test pass?

Reply:
This was a typo in the test, it should return 10